keyboards/macroboard/keymaps/default/keymap.c

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2021 Michał Szczepaniak
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = {
+        { KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5 },
+        { KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T},
+        { KC_ESC, KC_A, KC_S, KC_D, KC_F, KC_G},
+        { KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B},
+        { KC_LCTL, KC_LGUI, RGB_TOG, KC_LALT, RESET, KC_SPC}
+    },
```

Comment:
This should use a layout macro, rather than the raw matrix. 
```suggestion
    [0] = LAYOUT(
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,
        KC_ESC,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,
        KC_LCTL, KC_LGUI, RGB_TOG, KC_LALT, RESET,   KC_SPC
    )
```